recipes/snpeff/snpeff.py

```diff
@@ -95,10 +102,13 @@ def main():
 
     jar_path = os.path.join(jar_dir, jar_file)
 
-    java_args = [java] + mem_opts + prop_opts + [jar_arg] + [jar_path] + pass_args
+    java_args = [java] + mem_opts + prop_opts + [jar_arg] + [jar_path] + pass_args + disable_phone_home_opts
 
-    sys.exit(subprocess.call(java_args))
+    if is_debug:
+        sys.exit(print(java_args))
```

Comment:
Normally I would expect `--debug` to also run the command, just with more logging. Maybe `--dry-run` or something like that?

Reply:
Nice catch, renamed to `--dry-run`